src/components/Calculations/Calculations.js

```diff
@@ -1,9 +1,32 @@
-import { BillContext } from '../TipCalculatorContext/TipCalculatorContext';
-import { useContext } from 'react';
-
 ///tip per person = bill amount x tip percentage / number of people
 
-export const tipPerPerson = (tip, noOfPersons, bill) => {
-  const tipPerPerson = (bill * tip) / noOfPersons;
-  return tipPerPerson;
+export const tipPerPerson = (tip, customTip, noOfPersons, bill) => {
+  console.log('Inputs:', { tip, customTip, noOfPersons, bill });
+  const userTip = tip !== 0 ? tip / 100 : customTip / 100;
```

Comment:
You could check if `tip` and `noOfPersons` are both equal to 0 before doing any calculations